erpnext/setup/setup_wizard/operations/install_fixtures.py

```diff
@@ -379,6 +380,15 @@ def add_uom_data():
 				"value": d.get("value")
 			}).insert(ignore_permissions=True)
 
+def add_status_data():
+	status = json.loads(open(frappe.get_app_path("erpnext", "setup", "setup_wizard", "data", "status.json")).read())
```

Comment:
- Create a list of the statuses rather than creating the json

Reply:
I have recorded 32 status for existing erpnext so it's better to add it in the .json